docs/src/reference/2_blmo_build.md

```diff
@@ -0,0 +1,24 @@
+# LMO 
+
+This section contains information about interface blmo_interface.jl , build_lmo.jl and time_tracking_lmo.jl
```

Comment:
This section describes the general interface to the Bounded Linear Minimization Oracle and the preimplemented types of BLMO.